leveler/leveler.py

```diff
@@ -3054,7 +3066,13 @@ async def draw_levelup(self, user, server):
             image = await r.content.read()
         level_background = BytesIO(image)
         level_avatar = BytesIO()
-        await user.avatar_url_as(format=AVATAR_FORMAT).save(level_avatar, seek_begin=True)
+        try:
+            await user.avatar_url_as(format=AVATAR_FORMAT).save(level_avatar, seek_begin=True)
+        except (discord.NotFound, discord.HTTPException):
```

Comment:
```suggestion
        except discord.HTTPException:
```
~~*is there a way to add one same comment to lines that not consecutive on github?*~~
discord.NotFound is subclass of discord.HTTPException